### ISSUE TITLE
Missing .pkl files for the continuous_variables /discretize.py

### ISSUE THREAD

**Issue by mirzafarhan7** [btheodorou99/HALO_Inpatient#3]
Hi, I am interested in generating time series data with your code.  So, when I was looking at the continuous variables folder, I found that some .pkl files are missing. Could you provide the code for generating them in the repo or email me at: mfarhan@udel.edu.?

- idToLab = pickle.load(open('data/idx_to_lab.pkl', 'rb'))
- labToNumber = {l: i for (i,l) in enumerate(pickle.load(open('data/id_to_channel.pkl', 'rb')))}
- isCategorical = pickle.load(open('data/is_categorical_channel.pkl', 'rb'))
- beginPos = pickle.load(open('data/begin_pos.pkl', 'rb'))
- possibleValues = pickle.load(open('data/possible_values.pkl', 'rb'))
- variableRanges = pickle.load(open('data/variable_ranges.pkl', 'rb'))

Also, is it possible to generate both discrete and time series data at the same time ?

**Comment by btheodorou99** [comment]
My apologies for the delayed response. Most of those pickle files are either adapted from the same MIMIC Benchmark repository as the phenotypes or generated directly in some simple way (e.g. idToLab is basically just enumerating through the labs and mapping number to lab name). I have directly added those files to the repository. I have also added an additional dataset generation script to build the training dataset from scratch (as it is slightly different from the core dataset), note that this requires having certain preprocessing from the MIMIC Benchmark repository done as it make the lab processing much faster. Finally, I have also emailed you the script from the other issue that you asked about. As for more general time series and discrete data, the architecture can handle it, but adjustments need to be made depending on the desired data format. Continuous variables can be included with each visit to capture co-occurring variables such as time gaps or common labs. However, for full time series (e.g. a patient has a series of lab values for each stay) adjustments need to be made to the data representation as we have done in these scripts (for our experiments, we did this by having two types of visits, full hospital stay visits with the medical codes and then subsequent "time series visits" with just lab values that appear between hospital stay visits).
